editor/modes/visual-editor/block.js

```diff
@@ -129,7 +128,8 @@ class VisualEditorBlock extends wp.element.Component {
 					}
 				},
 				...blocksWithTheSameType.slice( 1 )
-			]
+			],
+			{ offset: -1 }
```

Comment:
I plan to resolve the offset:-1 differently in the future, but that will require some more changes.